listenbrainz/webserver/templates/navbar.html

```diff
@@ -21,26 +21,26 @@
     </a>
```

Comment:
Is this template file still in use?

Reply:
Yes, this template is still being used by `auth.html` for the AudioScrobbler API https://github.com/metabrainz/listenbrainz-server/blob/cd0fcf958441084bd341add43b079f1722f2cb22/listenbrainz/webserver/views/api_compat.py#L44-L78

Reply:
Update: The Auth page has also been migrated to SPA. Now Navbar.html is being used in `error/{code}.html` files.